update dynamic sampling page and move sampling page (#5851)
getsentry/sentry-docs: src/docs/product/data-management-settings/dynamic-sampling/index.mdx

## Dynamic Sampling & SDK Sampling

<Note>

> When we talk about SDK sample rates in this section, we're only referring to your [transaction sample rate](/platform-redirect/?next=/configuration/sampling/#sampling-transaction-events) (or `tracesSampleRate`), not your [error sample rate](/platform-redirect/?next=/configuration/sampling/#sampling-error-events).

Author: ```suggestion
When we talk about SDK sample rates in this section, we're only referring to your transaction sample rate (or `tracesSampleRate`), not your [error sample rate](/platform-redirect/?next=/configuration/sampling/%23sampling-error-events).
```

</Note>

Dynamic sampling and your SDK sampling configuration act independently of each other, but our dynamic sampling works best when you send us as much data as you can. The more complete a picture of your application we have, the more accurately we can monitor your application health, detect problems faster, surface issues, and make better decisions about data retention on your behalf.

> If it’s feasible, send us 100% of your data (that is, set your [tracesSampleRate](/platform-redirect/?next=/performance/) in the SDK to 1.0). If you can’t do so initially, the system will operate on what it can see and when you purchase more transaction volume, we’ll adapt automatically. As a bonus, your per-transaction pricing goes down accordingly. Learn more in [Why Dynamic Sampling](/product/data-management-settings/dynamic-sampling/why-dynamic-sampling/)

Author: ```suggestion
If it’s feasible, send us 100% of your data (that is, set your [tracesSampleRate](/platform-redirect/?next=/performance/) in the SDK to 1.0). If you can’t do so initially, the system will operate on what it can see and when you purchase more transaction volume, we’ll adapt automatically. As a bonus, your per-transaction pricing goes down accordingly. Learn more in [Benefits of Dynamic Sampling](/product/data-management-settings/dynamic-sampling/why-dynamic-sampling/).
```

> If it’s feasible, send us 100% of your data (that is, set your [tracesSampleRate](/platform-redirect/?next=/performance/) in the SDK to 1.0). If you can’t do so initially, the system will operate on what it can see and when you purchase more transaction volume, we’ll adapt automatically. As a bonus, your per-transaction pricing goes down accordingly. Learn more in [Benefits of Dynamic Sampling](/product/data-management-settings/dynamic-sampling/why-dynamic-sampling/).

Author: ```suggestion
If it’s feasible, send us 100% of your data (that is, set your [tracesSampleRate](/platform-redirect/?next=/performance/) in the SDK to 1.0). If you can’t do so initially, the system will operate on what it can see and when you purchase more transaction volume, we’ll adapt automatically. As a bonus, your per-transaction pricing goes down accordingly. Learn more in [Benefits of Dynamic Sampling](/product/data-management-settings/dynamic-sampling/benefits-dynamic-sampling/).
```

> If it’s feasible, send us 100% of your data (that is, set your [tracesSampleRate](/platform-redirect/?next=/performance/) in the SDK to 1.0). If you can’t do so initially, the system will operate on what it can see and when you purchase more transaction volume, we’ll adapt automatically. As a bonus, your per-transaction pricing goes down accordingly. Learn more in [Benefits of Dynamic Sampling](/product/data-management-settings/dynamic-sampling/benefits-dynamic-sampling/).

Author: ```suggestion
If it's feasible, send us 100% of your data (that is, set your [tracesSampleRate](/platform-redirect/?next=/performance/) in the SDK to 1.0). If you can't do so initially, the system will operate on what it can see and when you purchase more transaction volume, we'll adapt automatically. As a bonus, your per-transaction pricing goes down accordingly. Learn more in [Benefits of Dynamic Sampling](/product/data-management-settings/dynamic-sampling/benefits-dynamic-sampling/).
```
